src/azure-cli/azure/cli/command_modules/resource/tests/latest/management_group_level_template.json

```diff
@@ -43,73 +43,13 @@
       "type": "Microsoft.Authorization/policyAssignments",
       "name": "location-lock",
       "apiVersion": "2016-12-01",
-      "dependsOn": [ "policy2" ],
+      "dependsOn": [
+        "policy2"
+      ],
       "properties": {
         "scope": "[variables('mgScope')]",
         "policyDefinitionId": "[concat(variables('mgScope'), '/providers/', 'Microsoft.Authorization/policyDefinitions/', 'policy2')]"
       }
-    },
```

Comment:
May I know why we remove deployment resource below?

Reply:
Because it will cause an internal error of the Service, but there is no specific error message (only service internal error), the Service seems to have an unknown problem. I don't think these resources affect the main logic of the test deployment, so I removed them first.

Reply:
Resolved